qiskit/transpiler/passes/mapping/layout_score.py

```diff
@@ -0,0 +1,49 @@
+# -*- coding: utf-8 -*-
+
+# This code is part of Qiskit.
+#
+# (C) Copyright IBM 2019.
+#
+# This code is licensed under the Apache License, Version 2.0. You may
+# obtain a copy of this license in the LICENSE.txt file in the root directory
+# of this source tree or at http://www.apache.org/licenses/LICENSE-2.0.
+#
+# Any modifications or derivative works of this code must retain this
+# copyright notice, and modified files need to carry a notice indicating
+# that they have been altered from the originals.
+
+"""A pass to evaluate how good the layout selection was.
+No CX direction is considered.
+Saves in property_set[`layout_score`] the sum of distances for each circuit CX.
+The lower the number, the better the selection.
+Therefore, 0 is a perfect layout selection.
+"""
+
```

Comment:
Can the docstrings be updated to follow #3332 ?

Reply:
a5d1a4a3 should cover everything... let me know if i missed something.